### ISSUE TITLE
Add needed BIOS settings

### ISSUE THREAD

**Issue by 123marvin123** [drasbeck/macos-thinkpad-t430#11]
Some BIOS settings are mandatory for macOS to boot / install properly. It can be a very frustrating process if one does not know the correct settings.

The README file should be updated with the necessary BIOS settings.

**Comment by drasbeck** [comment]
Yes, it makes sense. Let's compare settings. I will note mine down soon, I hope you'll do the same.

**Comment by 123marvin123** [comment]
I'll check today.

**Comment by lyiriyah** [comment]
any update on the req'd bios settings?

**Comment by 123marvin123** [comment]
![IMG_0455](https://user-images.githubusercontent.com/7266447/56493524-acc40680-64ef-11e9-9a23-942c4e8ce1e6.jpeg)
![IMG_0453](https://user-images.githubusercontent.com/7266447/56493525-acc40680-64ef-11e9-8067-e801adc2aa12.jpeg)
![IMG_0452](https://user-images.githubusercontent.com/7266447/56493526-acc40680-64ef-11e9-9aba-85756fc7c5a4.jpeg)
![IMG_0456](https://user-images.githubusercontent.com/7266447/56493527-ad5c9d00-64ef-11e9-8079-e9630d460d5a.jpeg)
![IMG_0454](https://user-images.githubusercontent.com/7266447/56493528-ad5c9d00-64ef-11e9-91da-7598ce955fd2.jpeg)
![IMG_0460](https://user-images.githubusercontent.com/7266447/56493529-ad5c9d00-64ef-11e9-959a-ef8f56ff6990.jpeg)

Not all of the settings are necessary. Hope it helps.

**Comment by lyiriyah** [comment]
i found these for an x220 install off of [this site](http://x220.mcdonnelltech.com/)

- Restart > Load Setup Defaults
- Config > Power > Power On with AC Attach > Disabled
* Config > Serial ATA (SATA) > AHCI
* Security > Memory protection > Execution Prevention > Enabled
* Startup > UEFI/Legacy Boot > Both

**Comment by lyiriyah** [comment]
oh thanks
we posted at the same time

**Comment by Legetzu** [comment]
Intel AMT is recommended to keep on. It gives more hardware info for the system. Unfortunately in my T430 it's permanently disabled. With AMT in many hackintoshes you can fix issues with QE/CI. But if your hackintosh works fine without it being on I'd keep it disabled.

**Comment by 123marvin123** [comment]
Enabling Intel AMT does not change anything on my hackintosh, I don't see / feel any difference.

**Comment by Legetzu** [comment]
Then it seems it's unnecessary for T430. RehabMan has said that it needs to be on to inject IMEI but nowadays it seems it's not that important.

**Comment by drasbeck** [comment]
I don't see the AMT option in my BIOS, so I guess I don't have to worry about whether or not to disable it. =)

**Comment by Legetzu** [comment]
@drasbeck That's odd. I just thought the AMT setting might be helpful for some people because I've heard it fixes some graphic related issues on some setups.
